lib/u3d/installer.rb

```diff
@@ -86,6 +86,21 @@ def installed_sorted_by_versions
       return [] if list.empty?
       list.sort { |a, b| UnityVersionComparator.new(a.version) <=> UnityVersionComparator.new(b.version) }
     end
+
+    protected
+
+    def extra_installation_paths
+      return [] if ENV['U3D_EXTRA_PATHS'].nil?
+      ENV['U3D_EXTRA_PATHS'].strip.split(';')
+    end
+
+    def find_installations_with_path(default_path, *postfix)
```

Comment:
Not super fond of the use of variable arguments with a first argument here. No better idea yet.

Reply:
Maybe a named argument for the first one?

Reply:
Maybe. And maybe we should call it default_root_path?